### DISCUSSION TITLE
Error at startup

### DISCUSSION THREAD

**Discussion by karamzyatina** [plan-player-analytics/Plan#2572] [Q&A] [answered]
![opera_Rjywf51gjs](https://user-images.githubusercontent.com/67735224/187017033-44bfb87b-1ba0-4c1e-8f5a-7fe235f937d6.png)
When running PLAN on the latest version of Velocity, it outputs an error to the console:

**Comment by AuroraLS3** [comment, marked as answer]
This is solved in the latest dev release https://github.com/plan-player-analytics/Plan/releases/tag/5.5.1833

**Reply by karamzyatina** [reply]
thanks
